frameworks/keyed/yew/src/lib.rs

```diff
@@ -201,7 +201,7 @@ impl Component for Jumbotron {
             <div class="jumbotron">
                 <div class="row">
                     <div class="col-md-6">
-                        <h1>{ "Yew" }</h1>
+                        <h1>{ "Yew-Hooks" }</h1>
```

Comment:
I believe this should be in the `yew-hooks` implementation, not the `yew` implementation